Sources/Data Model/Audience/AttributeValue.swift

```diff
@@ -182,6 +186,32 @@ extension AttributeValue {
         return currentDouble < targetDouble
     }
     
+    func isLessOrEqual(than target: Any, condition: String = "", name: String = "") throws -> Bool {
+        return try isLess(than: target, condition: condition, name: name) || isExactMatch(with: target, condition: condition, name: name)
+    }
+    
+    func isSemanticVersionEqual(than target: SemanticVersion) throws -> Bool {
+        return (self.stringValue as SemanticVersion).compareVersion(targetedVersion: target) == 0
+    }
+
+    func isSemanticVersionGreater(than target: SemanticVersion) throws -> Bool {
+        return (self.stringValue as SemanticVersion).compareVersion(targetedVersion: target) > 0
+    }
+
+    func isSemanticVersionLess(than target: SemanticVersion) throws -> Bool {
+        return (self.stringValue as SemanticVersion).compareVersion(targetedVersion: target) < 0
+    }
+
+    func isSemanticVersionGreaterOrEqual(than target: SemanticVersion) throws -> Bool {
+        return try isSemanticVersionGreater(than: target) ||
```

Comment:
@thomaszurkan-optimizely What about 
`(self.stringValue as SemanticVersion).compareVersion(targetedVersion: target)` >= 0
and same for less than <=
we can save one call when version is equal.